doc/index.qdoc

```diff
@@ -0,0 +1,78 @@
+/****************************************************************************************
+**
+** Copyright (C) 2021 Open Mobile Platform LLC
+** All rights reserved.
+**
+****************************************************************************************/
+
+/*!
+\page index.html
+\title Sailfish WebView Component
```

Comment:
I'd actually drop that "Component" as well.

Reply:
Have done so.